Wrote 10 nouns in and their actions in plain english and in pseudo code

KomorebiAudio opened the pull request:
Hey man, hopefully this comes through right this time.

KomorebiAudio committed:
Wrote 10 nouns in and their actions in plain english and in pseudo code

matkatmusic commented on main.cpp:
```diff
@@ -52,86 +52,88 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
-
-//  action 3:
-
+//  Noun: man
+//  action 1: the man shouts
+man.shout();
+//  action 2: the man falls
+man.fall();
+//  action 3: the man punches
+man.punch();
```
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

matkatmusic reviewed (changes requested):
see my previous notes

KomorebiAudio committed:
Changed lines 106-112 as per request from MatKat.

REPLACE the highlighted noun/action as follows:

come up with a new noun and 3 actions

pick a noun that needs camelCase for the name

pick actions that need camelCase for their name.
